docs/user-guide/station-credential-management.md

```diff
@@ -0,0 +1,61 @@
+# Basics Station credentials management
+
+Starting with 'Azure IoT Edge LoRaWAN Starter Kit' v2.0.0, the LoRaWan Network Server runs a WebSocket endpoint compatible with [LNS Protocol](https://doc.sm.tc/station/tcproto.html) from LoRa Basics™ Station.
+
+As it's possible to read in [official LoRa Basics™ Station documentation](https://doc.sm.tc/station/credentials.html) a Basics™ Station client needs some credentials to establish a secure connection to LNS compatible endpoints.
+
+## Server Authentication
+
+### Importing certificate for server authentication in LoRaWan Network Server module
+
+LoRaWan Network Server IoT Edge module allows to import, from disk, a certificate for server authentication in 'pkcs12' format (.pfx).
+
+Two environment variables need to be set for making this happen:
+
+- **LNS_SERVER_PFX_PATH**: It's the absolute path to the .pfx certificate in the IoT Edge module filesystem (i.e.: '/var/lorastarterkit/certs/lns.pfx')
+- **LNS_SERVER_PFX_PASSWORD** *(optional)*: needs to be set if the .pfx was exported with password
+
+As stated above, LNS_SERVER_PFX_PATH is a path inside the IoT Edge module filesystem.
+
+Assuming the .pfx file is located in a folder on the host OS at /mnt/lora/certs, you will need to 'bind' this path to one in the IoT Edge module itself. In order to do so:
+
+1. Log into your Azure Portal
+
+2. Identify the IoT Edge Device in IoT Hub
+
+3. Set the 'LoRaWanNetworkSrvModule' HostConfiguration to include a binding for the folder
+
+    ```json
+    "Binds":  [
+    "/mnt/lora/certs/:/var/lorastarterkit/certs/"
+    ]
+    ```
+
+Additional reference on this process can be found [here](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-access-host-storage-from-module?view=iotedge-2020-11).
```

Comment:
```suggestion
Additional information on this process can be found in the documentation - [Use IoT Edge device local storage from a module](https://docs.microsoft.com/azure/iot-edge/how-to-access-host-storage-from-module?view=iotedge-2020-11).
```
make it a habit to match the title of the target document when referencing links. Also if you reference links from the docs, I recommend removing the local specific part.